### PR TITLE
fix duplicate mail in mailing-list-subscription-guide_dev.md 

### PR DESCRIPTION
**Pull request by yanghesong**
Raw URL of mail https://github.com/apache/dubbo/issues/1393) (#948)

### CODE REVIEW

**Review by wangchengming666 — APPROVED** [review]
LGTM

**Review by horizonzy — COMMENTED** [review]
now dubbo already graduated, should not use incubator.

File: content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ subscription address.
 
 4. 如果您想取消订阅，只需发送一封邮件到dev-unsubscribe@dubbo.apache.org，收到回复后，请按其指定步骤执行。
 
+> 请注意：dev@dubbo.apache.org和dev@dubbo.incubator.apache.org都是有效的, 您可以订阅它们中的任何一个。
```
Comment (AlbumenJ) [inline content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md:84]: please remove `dev@dubbo.incubator.apache.org`

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md (hunk 1 above)
Comment (yanghesong) [reply content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md]: Done

File: content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ subscription address.
 
 4. 如果您想取消订阅，只需发送一封邮件到dev-unsubscribe@dubbo.apache.org，收到回复后，请按其指定步骤执行。
 
+> 请注意：dev@dubbo.apache.org是有效的。dev@dubbo.incubator.apache.org已经不再适用。
```
Comment (AlbumenJ) [inline content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md:84]: `适用` -> `使用`

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md (hunk 2 above)
Comment (yanghesong) [reply content/zh/contribution-guidelines/contributor/mailing-list-subscription-guide_dev.md]: Done

**Pull request merged**